demos/asp-net-core/Divergent.Sales.API.Host/ServiceHost.cs

```diff
@@ -1,27 +0,0 @@
-﻿using Microsoft.Owin.Hosting;
-using System;
-using System.Configuration;
-
-namespace Divergent.Sales.API.Host
-{
-    class ServiceHost
-    {
-        IDisposable webApp;
-
-        public void Start()
-        {
-            var baseAddress = ConfigurationManager.AppSettings["baseAddress"];
-
-            webApp = WebApp.Start<Startup>(url: baseAddress);
-            Console.WriteLine($"Divergent.Sales.API.Host listening on {baseAddress}");
```

Comment:
it's handy to have the `baseAddress` printed on the console, especially if something goes wrong or you want to demo live something else.

Reply:
OK, I'll put that back in.